Kontent.Ai.Management/Models/PreviewConfiguration/TypePreviewUrlPatternRequestModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Kontent.Ai.Management.Models.Shared;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Kontent.Ai.Management.Models.PreviewConfiguration;
+
+/// <summary>
+/// Represents content type preview URL patterns request model.
+/// </summary>
+public class TypePreviewUrlPatternRequestModel
```

Comment:
SDK does not use Request suffix. Also this model is identical to TypePreviewUrlPatternResponseModel. Can we unify it?

Reply:
removed and unified